README.md

```diff
@@ -43,17 +43,18 @@ Usage: autocannon [opts] URL
 
 Available options:
 
-  -c/--connections NUM  The number of concurrent connections to use
-  -p/--pipelining NUM   The number of pipelined requests to use
-  -d/--duration SEC     The number of seconds to run the autocannnon
-  -m/--method METHOD    The http method to use
-  -b/--body BODY        The body of the request
-  -i/--input FILE       The body of the request
-  -H/--headers K=V      The request headers
-  -B/--bailout NUM      The number of failures before initiating a bailout
-  -j/--json             Print the output as json
-  -l/--latency          Print all the latency data
-  -h/--help             Print this menu
+  -c/--connections NUM  The number of concurrent connections to use. default: `10`.
+  -p/--pipelining NUM   The number of pipelined requests to use. default: `1`.
+  -d/--duration SEC     The number of seconds to run the autocannnon. default: `10`.
+  -m/--method METHOD    The http method to use. default: `'GET'`.
+  -b/--body BODY        The body of the request. default: `undefined`.
+  -i/--input FILE       The body of the request. default: `undefined`.
+  -H/--headers K=V      The request headers. default: `undefined`.
+  -B/--bailout NUM      The number of failures before initiating a bailout. default: `undefined`.
+  -p/--progress         Display the progress bar. default: `true`.
```

Comment:
After the fiasco with the npm progress bar, some people might like this option? I just thought it might be nice to give users the option.